WordPress/WordPressTest/TestUtilities/JSONObject.swift

```diff
@@ -7,23 +7,34 @@ extension JSONObject {
 
     /// Loads the specified json file and returns a dictionary representing it.
     ///
-    /// - Parameter fileName: The full name of the json file to load.
+    /// - Parameter fileName: The name of the json file to load. The "json" file extension can be omitted.
     /// - Returns: A dictionary representing the contents of the json file.
-    static func loadFile(named fileName: String) throws -> JSONObject {
-        return try loadFile(
-            (fileName as NSString).deletingPathExtension,
-            type: (fileName as NSString).pathExtension
-        )
-    }
+    static func loadJSONFile(named fileName: String) throws -> JSONObject {
```

Comment:
Nit on the name here, if we're reading a file into an object I'd expect to see something like `fromJSONFile`, or even better, an initializer` like ` JSONObject(fromFileNamed: String)` – the static initializer is fine, but the extra level of indirection required to use the concise version could be IMHO replaced by an initializer and be more clear.

WDYT?

Reply:
Great suggestion! I'll make the change.

Reply:
A case of "_I wish I thought of that suggestion_" 😄